pwndbg/commands/got.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import subprocess
+import struct
+
+import pwndbg.commands
+import pwndbg.commands.checksec
+import pwndbg.enhance
+import pwndbg.chain
+import pwndbg.which
+
+from pwndbg.color import *
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
```

Comment:
Ok, but we have to agree about the organization of the code. In your hints now readelf is a separate module and not included in wrappers. How about this organization: pwndbg/pwndbg/wrappers/readelf.py, then a new wrapper will be pwndbg/pwndbg/wrappers/checksec.py etc...

@disconnect3d  @zachriggle  @anthraxx  feedbacks? :smiley: